lib/v2/jump/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'jumpvg.com': {
+        _name: 'jump app',
+        switch: [
+            {
+                title: '折扣清单',
+                docs: 'https://docs.rsshub.app/game.html#jump',
+                source: ['/'],
+                target: '/discount/switch',
```

Comment:
```suggestion
                target: '/jump/discount/switch',
```
[script-standard](https://docs.rsshub.app/joinus/script-standard.html) has a typo in radar rules.
[quick-start](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze) has the right format.